py3curl-wrapper/__init__.py

```diff
@@ -216,22 +218,22 @@ def __str__(self) -> str:
     DELETED = 204
     ERROR = 500
 
-    API_HEADERS = {'Accept': 'application/json'}
+    API_HEADERS: ClassVar = {"Accept": "application/json"}
```

Comment:
This should be ``ClassVar[dict[str, str]]``

Reply:
Awesome! Fixed in latest commit